doc/man/task-sync.5.in

```diff
@@ -30,10 +30,8 @@ the existing replica, and run `task sync`.
 
 .SS When to Synchronize
 
-Taskwarrior can perform a sync operation at every garbage collection (gc) run.
-This is the default, and is appropriate for local synchronization.
-
-For synchronization to a server, a better solution is to run
+Taskwarrior performs a sync operation when you run `task sync`.
```

Comment:
In Taskchampion, the word "operation" is a "term of art" and I think it's confusing to use a different connotation of that word, especially when we have the concept of `SyncOp`'s ("sync operations"). I would consider deleting this line because it isn't clear to me that it communicates anything beyond the obvious: the `sync` subcommand performs a sync.